server/src/controllers/SearchController.js

```diff
@@ -41,12 +44,15 @@ class SearchController {
       return res.status(200).json(results);
     }
 
-    const q1 = 'SELECT "Articles".id, "Articles"."userId", "Articles".title, "Articles".description,  "Articles".content, "Articles".slug,';
-    const q2 = ' "Articles"."featuredImg", "Articles"."createdAt", "Articles"."createdAt", "Articles"."updatedAt" FROM "Articles"';
-    const q3 = ' INNER JOIN "Users" ON "Articles"."userId" = "Users"."id" AND "Articles".published = \'TRUE\' WHERE "Users"."username" =';
-    const q4 = ` '${filters.author}' AND "Articles"."searchVectors" @@ to_tsquery('${searchTerm}')`;
-    const q5 = 'ORDER BY "Articles"."createdAt" DESC';
-    const query = `${q1}${q2}${q3}${q4}${q5}`;
+    const q1 = 'SELECT "Articles".id, "Articles"."userId", "Articles".title,';
+    const q2 = ' "Articles".description,  "Articles".content, "Articles".slug,';
+    const q3 = ' "Articles"."featuredImg", "Articles"."createdAt", ';
+    const q4 = '"Articles"."createdAt", "Articles"."updatedAt" FROM "Articles"';
+    const q5 = ' INNER JOIN "Users" ON "Articles"."userId" = "Users"."id" AND ';
+    const q6 = '"Articles".published = \'TRUE\' WHERE "Users"."username" =';
+    const q7 = ` '${filters.author}' AND "Articles"."searchVectors" @@ to_tsquery('${searchTerm}')`;
+    const q8 = 'ORDER BY "Articles"."createdAt" DESC';
+    const query = `${q1}${q2}${q3}${q4}${q5}${q6}${q7}${q8}`;
```

Comment:
@ajimae @65black  I believe we could have found a better way of doing this. Not sure right now.  I will go ahead to merge this now but please ensure that you discuss this with your TTL.